src/Nuget.PackageIndex.VisualStudio/ProjectKFilter.cs

```diff
@@ -43,8 +86,114 @@ public bool IsProjectSupported(string filePath)
                     // TODO Package manager console?
                     Debug.Write(string.Format("{0} \r\n {1}", e.Message, e.StackTrace));
```

Comment:
Well if you have an issue, you want to share it so it can get reported.

Also instead of  `e.Message` and `e.StackTrace` and formatting you can just do `e.ToString()` same effect less electrons on the screen :)